lib/ts/recipe/thirdpartypasswordless/components/themes/signInUp/passwordlessForm.tsx

```diff
@@ -0,0 +1,170 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
Why do we need to reimplement so much here? Let's discuss on a call.

Reply:
There are a few reasons:
- We need to override the header but it also depends on the state.
- We need to provide different style contexts based on the state
- We need to control how errors are shown.
I think there is a better solution that would also do away with the embedded feature components. Let's discuss this tomorrow.